src/Questions/Checkbox/__tests__/checkbox.test.js

```diff
@@ -18,7 +19,7 @@ const question = {
   }
 }
 
-it('can checked/unchecked', () => {
+it('can checked/unchecked and default checked', () => {
```

Comment:
I guess that would be a separate text for `defaultChecked` giving a question with a defaultChecked to true and also falso in the same test context

Reply:
this comment is fixed in the new commit : unit test for default checked = true and false
d406c7e

Reply:
Yes, exactly!